test/test_common.py

```diff
@@ -47,6 +48,7 @@
         RobustMissingIndicator(),
         RobustStandardScaler(),
         ThresholdOneHotEncoder(),
+        # WOEEncoder(),
```

Comment:
Please uncomment this

Reply:
this won't work as per PR comment, please advise